packages/rum/test/specs/apm-base.spec.js

```diff
@@ -386,36 +386,56 @@ describe('ApmBase', function () {
     expect(loggingService.warn).toHaveBeenCalledWith('RUM agent is inactive')
   })
 
-  it('should log errors when config is invalid', () => {
-    const loggingService = serviceFactory.getService(LOGGING_SERVICE)
-    spyOn(loggingService, 'warn')
+  describe('invalid config', () => {
     const logErrorSpy = spyOn(loggingService, 'error')
```

Comment:
There loggingService reference doesn't exist there yet, and the build fails. A possible way to approach this could be to isolate the loggingService related stuff in one place: (just an example!)

```
 let loggingService

    beforeEach(() => {
      loggingService = serviceFactory.getService(LOGGING_SERVICE)
      spyOn(loggingService, 'warn')
      spyOn(loggingService, 'error')
    })
```